sdk/OoyalaSkinSDK/videoView.js

```diff
@@ -90,15 +94,23 @@ var VideoView = React.createClass({
   },
 
   onSocialButtonPress: function(socialType){
+    console.log("on onSocialButtonPress")
     this.props.onSocialButtonPress(socialType);
   },
 
   handlePress: function(name) {
+    //console.log("in handle press");
+    console.log("in handle press"+name);
+    // console.log("in handle press play pause"+BUTTON_NAMES.PLAY_PAUSE);
+    // console.log("in handle press auto hide"+BUTTON_NAMES.RESET_AUTOHIDE);
+    // console.log("in handle press props play"+this.props.isPlay);
```

Comment:
remove comments, use Log.log, more professional ("In handlePress. Button: " + name)